lib/cluster/etcd_test.go

```diff
@@ -666,3 +666,12 @@ func TestEtcdKeyDataPublisher_Publish_timeout(t *testing.T) {
 		})
 	}
 }
+
+func TestEtcd_GetStorageKey(t *testing.T) {
+	key, err := cluster.GetStorageKey("/foo", "/foo/config/all")
+	require.NoError(t, err)
+	require.Equal(t, "all", key)
+
+	_, err = cluster.GetStorageKey("/foo", "/bar/config/all")
+	require.EqualError(t, err, "source must begin with: /foo/config/")
+}
```

Comment:
There are two different test cases in fact, not a single test.
```suggestion
func TestGetStorageKey(t *testing.T) {
	key, err := cluster.GetStorageKey("/foo", "/foo/config/all")
	require.NoError(t, err)
	require.Equal(t, "all", key)

	_, err = cluster.GetStorageKey("/foo", "/bar/config/all")
	require.EqualError(t, err, "source must begin with: /foo/config/")
}
```

Reply:
Separated.